nx/lib/core/cache.ml

```diff
@@ -0,0 +1,16 @@
+(* cache.ml *)
```

Comment:
I'd suggest naming it Cache_dir or Xdg_cache to not be confused with other kind of cache.